task/06-conditions-n-loops-tasks.js

```diff
@@ -251,7 +309,23 @@ function reverseInteger(num) {
  *   4916123456789012 => false
  */
 function isCreditCardNumber(ccn) {
-    throw new Error('Not implemented');
+    const stringCCN = [...ccn.toString()].reverse();
+
+    const modifiedCCN = stringCCN.map((value, index) => {
+        if (index % 2) {                     //  double every second number
```

Comment:
`index % 2)` is not safe condition. It will be `1` or `0`, not `true` or `false`,